### PR TITLE
go: cleanup

### PR DESCRIPTION
**Pull request by charlierudolph**
from comments on #117

### CODE REVIEW

File: go/exocom-mock/exocom-mock.go [hunk 1]
```diff
@@ -28,42 +30,62 @@ func New() *ExoComMock {
 }
 
 // Close takes this ExoComMock instance offline
-func (exoCom *ExoComMock) Close() error {
-	return exoCom.server.Close()
+func (e *ExoComMock) Close() error {
+	return e.server.Close()
```
Comment (alexdavid) [inline go/exocom-mock/exocom-mock.go]: 👍

File: go/exocom-mock/exocom-mock.go [hunk 2]
```diff
@@ -28,42 +30,62 @@ func New() *ExoComMock {
...
 
-// Reset nils the socket and clears all received messages
-func (exoCom *ExoComMock) Reset() {
-	exoCom.socket = nil
-	exoCom.ReceivedMessages = []structs.Message{}
+// Reset closes and nils the socket and clears all received messages
+func (e *ExoComMock) Reset() {
+	if e.HasConnection() {
+		err := e.socket.Close()
+		if err != nil {
```
Comment (alexdavid) [inline go/exocom-mock/exocom-mock.go]: What do you think about joining these lines?

```go
if err := e.socket.Close(); err != nil {
```

File: go/exorelay/main.go [hunk 3]
```diff
@@ -35,25 +35,25 @@ func New(config Config) *ExoRelay {
 }
 
 // Connect brings an ExoRelay instance online
-func (exoRelay *ExoRelay) Connect() error {
-	socket, err := websocket.Dial("ws://"+exoRelay.config.Host+":"+exoRelay.config.Port, "", "origin:")
+func (e *ExoRelay) Connect() error {
+	socket, err := websocket.Dial("ws://"+e.config.Host+":"+e.config.Port, "", "origin:")
```
Comment (alexdavid) [inline go/exorelay/main.go]: Is there a reason why `e.config.Port` is a string here?

File: go/exocom-mock/exocom-mock.go (hunk 2 above)
Comment (charlierudolph) [inline go/exocom-mock/exocom-mock.go]: I like that idea when the function call is short but we have many other similar instances where the function call is long. For consistency, I'd prefer to leave it like this

File: go/exorelay/main.go (hunk 3 above)
Comment (charlierudolph) [inline go/exorelay/main.go]: no. I'll convert it to an int.

**Comment by charlierudolph** [comment]
merged in 25dbacf

**Pull request closed without merging**